### ISSUE TITLE
No Corners Option

### ISSUE THREAD

**Issue by bnice5000** [malsup/corner#6]
I am not so sure if this is an issue so much as a feature request.  For inner elements in list the corners module rounds all the list items.  Is there a way to set a "no corners" options.  This way subsequent elements that would normally be affected by the corner element could be unset. An example might be:

<pseudo-code>
$("#element").corner("nocorners");
</pseudo-code>

**Comment by malsup** [comment]
What you described is not how the plugin works.  It only corners the elements selected by the jQuery selector.  If you have an example that shows otherwise then please post a link.

**Comment by bnice5000** [comment]
I actually rectified the issue I was having and then forgot to post back.  I have a dynamically sized site.  When a user, using IE, resized the window the corners went haywire.  I found the uncorner method.  I was able to implement it in such a way that it fixed my issue.  I have included the code below.

function IE_corners_fix()
{
  $("#sidebar-left-inner").uncorner("br");
  $("#sidebar-left-inner").corner("br");
  $("#sidebar-left-inner a.parent1").parent().corner("bl tl");  
}  

$(window).resize(function(){
  IE_corners_fix();
});
